sleepecg/feature_extraction.py

```diff
@@ -471,13 +471,20 @@ def preprocess_rri(
     -------
     np.ndarray
         The cleaned RRI series.
+
+    Examples
+    --------
+    Mask RR intervals outside the range of 0.4 to 2 seconds (= 30 to 150 bpm)
+
+    >>> from sleepecg import preprocess_rri
+    >>> preprocess_rri([0.5, 0.2, 0.8, 2.5, 0.6], min_rri=0.4, max_rri=2)  # noqa
+    array([0.5, nan, 0.8, nan, 0.6])
     """
-    invalid_rri = np.zeros_like(rri, dtype=bool)
+    rri = np.asarray(rri, dtype=float)
```

Comment:
```suggestion
    rri = np.array(rri, dtype=float)
```
[`np.asarray`](https://numpy.org/doc/stable/reference/generated/numpy.asarray.html) doesn't perform a copy if the input matches the desired dtype and order, so I think [`np.array`](https://numpy.org/doc/stable/reference/generated/numpy.array.html) would be the safest option here